### ISSUE TITLE
Create proper package.json

### ISSUE THREAD

**Issue by ankon** [Collaborne/cordova-gallery-api#1]
Since Cordova 7 the CLI uses cordova-fetch, which requires a `package.json` to exist.

This should be created using `plugman createpackagejson $PWD`, but due to https://github.com/apache/cordova-lib/pull/597 this doesn't work right now.

**Comment by ankon** [comment]
This is actually becoming critical: cordova 8 will seemingly remove the legacy parts that fetch using (a horribly outdated) npm, so we need to resolve this quickly: https://issues.apache.org/jira/browse/CB-9626

**Comment by orgotech-robin** [comment]
Im trying to install like the new instructions you added in the readme:
`cordova plugin add https://github.com/Collaborne/cordova-gallery-api.git --save --nofetch`

But without success.
Error:
```Failed to fetch plugin https://github.com/Collaborne/cordova-gallery-api.git via registry.
Probably this is either a connection problem, or plugin spec is incorrect.
Check your connection and plugin name/version/URL.
Error: npm: Command failed with exit code 1 Error output:
npm ERR! premature close

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/XXX/.npm/_logs/2018-12-18T14_45_10_899Z-debug.log```

Is there any solution to this atm?

**Comment by ankon** [comment]
@ronnyroeller can you also update the README instructions now? 

If I have to guess: We now require Cordova 8 and the use of npm for fetching, i.e. `--no-fetch` needs to go?

As to @orgotech-robin's comment: That is likely exactly what this issue was about, so it should be fixed now if I understand c17e227 correctly.
